app/assets/javascripts/global.js

```diff
@@ -28,6 +28,15 @@ $(document).ready(function() {
     $( ".timepicker" ).datetimepicker({
       format: 'h:mm A'
     });
+
+    //Adds listeners to select inputs to toggle disclaimers on rental schedule page
+    $(document.getElementsByName("rental[item_type_id]")).each(function() {
+      $(this).change(function () {
+        $(this).closest("form").find(".disclaimer").each(function() {
+          $(this).toggleClass("disclaimer-hidden")
+        })
+      })
+    })
```

Comment:
@phantompatrol Hm, I tried a few different methods with closest("disclaimer") but it doesn't seem to return anything. I could look into that further? And no, selecting the same option again doesn't trigger the change 😃  On a sidenote I'm not sure why the travis build failed..
And aw the gif is gone 😢